docs/1.guide/01.index.md

```
@@ -83,6 +83,25 @@ bun run server.mjs
+| Runtime            | Types package               |
+| ------------------ | --------------------------- |
+| Node.js            | `@types/node`               |
+| Deno               | `@types/deno`               |
+| Bun                | `@types/bun`                |
+| Cloudflare Workers | `@cloudflare/workers-types` |
+| AWS Lambda         | `@types/aws-lambda`         |
+| Service Worker     | `@types/serviceworker`      |
+
+> [!NOTE]
+> `@types/deno` references DOM types, so pair it with `lib: ["esnext", "dom"]`.
+
```

markdownlint-cli2 (0.23.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)